### PR TITLE
Clean up upgrade test

### PR DESCRIPTION
**Pull request by mboukhalfa**
The PR cleans the unneeded tasks in the upgrade test

### CODE REVIEW

File: vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml [hunk 1]
```diff
@@ -1,52 +1,8 @@
...
-          strategy:
-            rollingUpdate:
-              maxSurge: 1
-              maxUnavailable: 1
-
-  - name: Untaint all nodes
-    ansible.builtin.command: kubectl taint nodes --all node-role.kubernetes.io/master-
-    environment:
-      KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    ignore_errors: yes
```
Comment (mboukhalfa) [inline vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml:24]: The Untaint here does not provide any useful addition, we have another untaint task after upgrading the controlplane to allow KCP to tolerate worker pods while that single worker is upgrading

**Comment by kashifest** [comment]
Thanks @mboukhalfa 
/hold cancel

**Pull request merged**